openlibrary/plugins/openlibrary/lists.py

```diff
@@ -62,8 +62,17 @@ def get_seed_info(doc):
 
 @public
 def get_list_data(list, seed, include_cover_url=True):
+    list_items = []
+    for s in list.get_seeds():
```

Comment:
@mekarpeles to check whether any caching here.

@jimchamp: may have performance issues if list has many seeds?

Reply:
Noting that this section isn't properly feature-flagged.  `list_items` should only be populated and added to list data `d` if `my_books_dropper` feature is enabled.

Reply:
> @jimchamp: may have performance issues if list has many seeds?

Yes, on book pages.  This is only called once on search results pages, but several times in book pages.  The public `get_user_lists` function also calls this while iterating over a list.  Cleaning up the lists widget template a bit should help.